pl_examples/basic_examples/autoencoder.py

```diff
@@ -13,17 +13,20 @@
 # limitations under the License.
 
 from argparse import ArgumentParser
+
 import torch
-from torch import nn
 import torch.nn.functional as F
+from torch import nn
 from torch.utils.data import DataLoader
-import pytorch_lightning as pl
 from torch.utils.data import random_split
 
-try:
+import pytorch_lightning as pl
+from pl_examples import TORCHVISION_AVAILABLE
+
+if TORCHVISION_AVAILABLE:
     from torchvision.datasets.mnist import MNIST
     from torchvision import transforms
-except ModuleNotFoundError:
+else:
```

Comment:
What about `transforms` in the `else` case case? that would be undefined

Reply:
I know that was a bit tricky, but we still have `torchvision` as requirements for examples...
basically, we can write a custom wrapper which does `toTensor()` or ignore this conversion

Reply:
fixed :]